### DISCUSSION TITLE
Will mackup honour XDG_CONFIG_HOME for .mackup.cfg ?

### DISCUSSION THREAD

**Discussion by andrewcrook** [lra/mackup#1962] [Q&A]
Will mackup honour XDG_CONFIG_HOME for .mackup.cfg ?

if so would this be  XDG_CONFIG_HOME/mackup.cfg
or  XDG_CONFIG_HOME/mackup/mackup.cfg

**Comment by joshmedeski** [comment]
I don't think it honors it right now, but I'd like to support `~/.config/mackup/mackup.cfg` and even move the custom applications to `~/.config/mackup/apps/*.cfg` (or something similar).

**Comment by NorthIsUp** [comment]
i added a pr for this #1987
